### ISSUE TITLE
IndexError happen at a project with identity=true & uniquebodyparts in 2.2rc1

### ISSUE THREAD

**Issue by bobfromjapan** [DeepLabCut/DeepLabCut#1222]
**Describe the bug**

In 2.2RC1, when I try to run the training with a multi-animal project `identity=true` and some `uniquebodyparts`, I got an `IndexError`.

**Desktop (please complete the following information about your system):**
 - OS: Windows 10
 - DeepLabCut Version: '2.2rc1'
 - DeepLabCut mode: multi-animal tracking

**To Reproduce**
1. make multi-animal project with `identity=true` *AND* having some `uniquebodyparts` in `config.yaml`
2. extract, labeling frames and create training dataset with GUI
3. trying train network, then got error like this:

<details><summary>TRACEBACK</summary><p>

```python
Exception in thread Thread-1:
Traceback (most recent call last):
  File "C:\Users\XXX\AppData\Local\Programs\Python\Python37\lib\threading.py", line 926, in _bootstrap_inner
    self.run()
  File "C:\Users\XXX\AppData\Local\Programs\Python\Python37\lib\threading.py", line 870, in run
    self._target(*self._args, **self._kwargs)
  File "C:\Users\XXX\pyenv\dlc_gpu_latest\lib\site-packages\deeplabcut\pose_estimation_tensorflow\train_multianimal.py", line 76, in load_and_enqueue
    batch_np = dataset.next_batch()
  File "C:\Users\XXX\pyenv\dlc_gpu_latest\lib\site-packages\deeplabcut\pose_estimation_tensorflow\dataset\pose_multianimal_imgaug.py", line 279, in next_batch
    joint_ids, batch_joints, data_items, sm_size, image_shape
  File "C:\Users\XXX\pyenv\dlc_gpu_latest\lib\site-packages\deeplabcut\pose_estimation_tensorflow\dataset\pose_multianimal_imgaug.py", line 222, in get_targetmaps_update
    joint_ids[i], [joints[i]], data_items[i], sm_size, scale
  File "C:\Users\XXX\pyenv\dlc_gpu_latest\lib\site-packages\deeplabcut\pose_estimation_tensorflow\dataset\pose_multianimal_imgaug.py", line 417, in compute_target_part_scoremap_numpy
    scmap[mask, person_id + num_joints] = 1
IndexError: index 10 is out of bounds for axis 2 with size 10
```

</p></details>

**Additional context Add any other context about the problem here.**

I got this error regardless of NetType; confirmed for ResNet50 and DLCRNet.

I thought `person_id + num_joints` is mismatching to `num_joints + num_idchannel` value that of used in scmap's initiation (`pose_multianimal_imgaug.py`). But I'm not too familiar with the new DeepLabCut, so I can't be sure.

This RC update looks amazing, and I'm really enjoying it!

I would love to try this `identity=true` feature. I would appreciate any advice!

Thank you in advance!

**Comment by AlexEMG** [comment]
Thanks for raising this, we have not needed this so far -- will be addressed with 2.2rc2

**Comment by bobfromjapan** [comment]
Thank you for responding!
I'll try it tomorrow.
